README.md

```diff
@@ -1,25 +1,22 @@
-# Initial setup
+# HDZero Goggle Firmware
 
-## 1. Extract supporting libraries and head files
+## Building in a devcontainer
 
-```
-~/hdz_goggle/code$ tar xzvf linux-4.9.tar.gz
-~/hdz_goggle/code$ cat media.tar.gzaa media.tar.gzab > media.tar.gz
-~/hdz_goggle/code$ tar xzvf media.tar.gz
-~/hdz_goggle/code$ tar xzvf toolchain.tar.gz
-```
+This repository supports the [vscode devcontainer](https://code.visualstudio.com/docs/devcontainers/containers) integration.  
+To get started install docker, vscode and the decontainer extension.  
+A [prompt](https://code.visualstudio.com/docs/devcontainers/create-dev-container#_add-configuration-files-to-a-repository) to reopen this repository in a container should appear.
 
-## 2. Setup cmake environment
+## Building locally
```

Comment:
I find this a bit confusing, as this is only a setup step and not actually building anything, so making that clear would be helpful. So the old naming was probably better here.

Reply:
...i hear you, what do you suggest? how about "Setup local environment"? i want to make sure that it is conwayed that is a either or deal.